src/test/coins_tests.cpp

```diff
@@ -558,16 +560,29 @@ BOOST_AUTO_TEST_CASE(ccoins_serialization)
     }
 }
 
+const static char DIRTY = CCoinsCacheEntry::DIRTY;
+const static char FRESH = CCoinsCacheEntry::FRESH;
+const static char NO_ENTRY = -1;
+
+struct CoinEntry {
```

Comment:
In commit "test: Migrate GetCoinsMapEntry to return MaybeCoin" (a902af6fb195fae97a89876b88d5034162ab8366)

May want to add "refactor" to commit subject or otherwise mention that this commit is not adding or removing any test coverage. Otherwise it's hard to know looking at the commit what it's intended to do.

Same comment also applies to commit "test: Remove remaining unbounded flags from coins_tests" (e4db49949c55470950a206172ee53f6baacf3767) and commit "test: Compact ccoins_access and ccoins_spend" (aa2f3139529c054b011a0f75ff314e6d63f0d977)

Reply:
Done